ndcube/ndcube.py

```diff
@@ -222,6 +223,10 @@ def dimensions(self):
         return u.Quantity(self.data.shape, unit=u.pix)
 
     @property
+    @deprecated(since='1.4.1',
+                message='NDCube.world_axis_physical_types will be removed in version 2.0. ' + \
+                        'Use NDCube.wcs.world_axis_physical_types or ' + \
+                        'NDCube.array_axis_physical_types instead.')
```

Comment:
you don't need line continuation inside brackets, or +.
```suggestion
                message=('NDCube.world_axis_physical_types will be removed in version 2.0. '
                        'Use NDCube.wcs.world_axis_physical_types or '
                        'NDCube.array_axis_physical_types instead.'))
```

Reply:
Thanks!